Makefile.am

```
@@ -0,0 +1,199 @@
+## Process this file with automake to produce Makefile.in
+#
+# CFITSIO Makefile.am
+#
+# Author: B.Irby (NASA/GSFC), based on the previous CFITSIO Makefile.in
+
+lib_LTLIBRARIES = libcfitsio.la
+
+# Shared library version suffixes generated by libtool, used below
+# for symbolic links:
+soname_version = @version_info@
+soname_version_lnx = ${soname_version}.0.0
+
+libcfitsio_ladir = $(pkgincludedir)
+libcfitsio_la_HEADERS =	fitsio.h fitsio2.h longnam.h drvrsmem.h \
+	cfortran.h f77_wrap.h region.h
```

This causes the headers to be installed into `${prefix}/include/cfitsio`, whereas before they were just installed to `${prefix}/include`. For backward compatibility, I suggest sticking with the old location:

```suggestion
include_HEADERS =	fitsio.h fitsio2.h longnam.h drvrsmem.h \
	cfortran.h f77_wrap.h region.h
```

If you decide to keep the new location, then please make sure to update cfitsio.pc.in.

---

Thanks for catching that, I prefer to retain backward compatibility.

---

Would you please apply that change so that I can test it again?